### PR TITLE
Translation update

### PR DESCRIPTION
**Pull request by Glandos**
This is a precondition to #335 and a try to solve #336

### CODE REVIEW

File: ihatemoney/translations/fr/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -7,289 +7,281 @@ msgid ""
...
+#: web.py:461
 #, python-format
-msgid "User '%(name)s' has been deactivated"
-msgstr "Le membre '%(name)s' a été désactivé"
+msgid ""
+"User '%(name)s' has been deactivated. It will still appear in the users "
+"list until its balance becomes zero."
+msgstr ""
+"Le membre '%(name)s' a été désactivé. Il continuera d'apparaître jusqu'à "
+"ce que sa balance devienne égale à zéro."
```
Comment (JocelynDelalande) [inline ihatemoney/translations/fr/LC_MESSAGES/messages.po]: balance -> solde

**Comment by JocelynDelalande** [comment]
@Glandos seems OK !

Is that quick for you to cleanup the PR log before I merge (like removing 0a9d16b and squashing 
893db0b  with 1947a5a ) ?

If no, no prob, I will merge as-is.

Could you also update CHANGELOG.md and CONTRIBUTORS file ?

Thanks by advance :-)

**Comment by Glandos** [comment]
I have to admit that my git/github skills are not high enough, to replace some already pushed commits by another one. I think I should open a new PR for that…
If you don't mind, I'll prefer to merge this as-is.

**Comment by JocelynDelalande** [comment]
> I have to admit that my git/github skills are not high enough, to replace some already pushed commits by another one. I think I should open a new PR for that…
> If you don't mind, I'll prefer to merge this as-is.

No prob for me ; I let you update CHANGELOG.md and CONTRIBUTORS files and then, I'll merge, ok ?

(FYI, for a next time, the process is based on `git rebase --interactive` and then `git push -f`)

**Comment by Glandos** [comment]
I'm already listed in `CONTRIBUTORS` :smile:

**Comment by JocelynDelalande** [comment]
> I'm already listed in CONTRIBUTORS smile

Oh sorry /o\.

Meeeerging !

**Pull request merged**